<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control de usuario</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Img/Users48.png</normaloff>../Img/Users48.png</iconset>
  </property>
  <widget class="QPushButton" name="btnSalir">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>SALIR</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Img/salir.png</normaloff>../Img/salir.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtUsuario">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPass">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnIniciar">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>ENTRAR</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Img/Entrar.png</normaloff>../Img/Entrar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Aharoni</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Contrase√±a : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Aharoni</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Usuario : </string>
   </property>
  </widget>
  <widget class="QLabel" name="lblError">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;
&lt;head/&gt;
&lt;body&gt;
&lt;img src=&quot;../Img/User24.png&quot; /&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;
&lt;head/&gt;
&lt;body&gt;
&lt;img src=&quot;../Img/Lock24.png&quot; /&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtUsuario</tabstop>
  <tabstop>txtPass</tabstop>
  <tabstop>btnIniciar</tabstop>
  <tabstop>btnSalir</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
